tests/testthat/test-config-log.R

```diff
@@ -1,6 +1,22 @@
 context("Constructing config log from bbi_config.json")
 
-check_config_ref <- function(log_df, run_nums, col_count) {
+# to minimize changes to the existing tests, we define the model and data status
+# for each of the models any particular test might need
+run_status <- dplyr::tribble(
```

Comment:
I'm not clear on why this is a variable that is passed into the function each time. It feels to me like a constant that should be in `setup-workflow-ref.R`. Am I missing something? Do you disagree?

Reply:
It's passed as an argument for referential transparency. We only call this function when we run the tests, so this should be fairly lightweight.

`run_status` is defined here rather than in `setup-workflow-ref.R` because 

- It's not used by any other test file.

- The further a value is defined from where it will be used, the harder it is to look it up. The values in this object are a direct consequence of the `setup()` call (copy this model here, copy that model there), so I wanted to keep the "expectation" close to its definition.

If you feel strongly, I can adopt the pattern you suggest.

Reply:
No I'm fine with that reasoning.